back-end/endpoints/endpoints.py

```diff
@@ -68,13 +68,17 @@ def get(self):
 #class for Login
 class Login(Resource):
     def post(self):
+       
         data = UserService.login(request.args.get('username'), request.args.get('password'))
-        return data, 200         
+        return data, 200        
 
 class CreateUser(Resource):
     def post(self):
-        data = UserService.createUser(request.args.get('username'), request.args.get('password'), request.args.get('email'), request.args.get('first'), request.args.get('last'))
-        return data, 200
+        if(UserService.isUserNameAvailable(request.args.get('username'))):
+            data = UserService.createUser(request.args.get('username'), request.args.get('password'), request.args.get('email'), request.args.get('first'), request.args.get('last'))
```

Comment:
technically i feel like this should be in a service but nbd/ probably not worth moving